lib/PHPExif/Adapter/FFprobe.php

```diff
@@ -94,6 +94,10 @@ public function getExifFromFile($file)
     {
         $mimeType = mime_content_type($file);
 
+        if ($mimeType === 'image/x-tga') {
+            $mimeType = 'video/mpeg';
```

Comment:
That's a funny one :) we should ignore these line for the code coverage calc

Reply:
It was a pain to figure out! Actually not so much on the php-exif side as in Lychee itself. I mean, who would've thought?! And yeah, I don't think we can simulate this one for coverage...